src/wp-includes/block-template-utils.php

```diff
@@ -249,7 +249,18 @@ function _get_block_templates_paths( $base_directory ) {
  *
  * @param string $template_type 'wp_template' or 'wp_template_part'.
  * @param string $slug          Template slug.
- * @return array|null Template.
+ * @return array|null {
+ *    Array with template metadata if $template_type is one of 'wp_template' or 'wp_template_part'.
+ *    null otherwise.
+ *
+ *    @type string $slug      Template slug.
+ *    @type string $path      Template file path.
+ *    @type string $theme     Theme slug.
+ *    @type string $type      Template type.
+ *    @type string $title     Optional. Template title.
+ *    @type string $area      Template area. Only for 'wp_template_part'.
+ *    @type array  $postTypes Optional. List of post types that the template supports. Only for 'wp_template'.
```

Comment:
```suggestion
 *    @type string[] $postTypes Optional. List of post types that the template supports. Only for 'wp_template'.
```

Ref: https://github.com/WordPress/gutenberg/blob/de94c50198e78c551f978849b69024d82b349617/schemas/json/theme.json#L2129-L2132